src/log/access_log.cc

```diff
@@ -244,21 +238,23 @@ HierarchyLogEntry::notePeerWrite()
 void
 HierarchyLogEntry::startPeerClock()
 {
-    if (!firstConnStart_.tv_sec)
-        firstConnStart_ = current_time;
+    if (peerTimer.running()) {
+        debugs(46, 5, "already running");
+        return;
+    }
+    peerTimer.resume();
```

Comment:
Since Stopwatch declaration has to be known to HierarchyLogEntry.h readers, let's inline this method (after it becomes trivial due to another change request). Same for stopPeerClock().